walkthroughs/week-2-server/examples/server-date/src/main/webapp/script.js

```diff
@@ -3,6 +3,8 @@ async function showServerTime() {
   const responseFromServer = await fetch('/date');
   const textFromResponse = await responseFromServer.text();
 
+  <p> textFromResponse </p>
```

Comment:
1. Wrong file (see PR-level comment)
2. Remove the `<p>` and `</p>`. This is a JS file and will crash when it tries to evaluate this line (since it's not valid syntax). HTML tags are not needed in the text used to set `dateContainer.innerText = textFromResponse;`. Think about it like this: whatever `dateContainer.innerText`  is set to will display as the **text inside the `date-container** element. So `dateContainer.innerText = "<p> something </p>"` will show `<p> textFromResponse </p>` (incl. the <p> tags).